### ISSUE TITLE
Chrome blocking attempts to create a WebMediaPlayer

### ISSUE THREAD

**Issue by BrandonMathis** [twilio/twilio-video-app-react#559]
**Describe the bug**
If two users (A and B) are in the same room & user B refreshes their page > 20 or so times the application will fall into an irrecoverable error state for user A. Chrome will no longer allow any additional `WebMediaPlayer` objects to be created.

```
[Intervention] Blocked attempt to create a WebMediaPlayer as there are too many WebMediaPlayers already in existence. See crbug.com/1144736#c27
``` 

[There is an ongoing conversation about this in chrome bug reports](https://bugs.chromium.org/p/chromium/issues/detail?id=1144736#c27). Chrome seems to cap the allowed number of `WebMediaPlayers` to 50 due to a memory leak issue 

**To Reproduce**
1. Have two users join the same room
2. Have one of those users refresh & join the room over and over again (takes about 20 or 30 times)
3. Eventually, the error above will appear in the chrome console. All future participants will have a black box for their video. Audio may or may not mount.

**Expected behavior**
Participants should be able to leave & re-join over and over again without impacting other participants.

**Screenshots**
Participants A (never refreshed, participants B rejoining over and over again causes this)
![Screen Shot 2021-07-27 at 1 01 28 PM](https://user-images.githubusercontent.com/362269/127196877-da5cb839-bb80-4eac-9961-e754ebfe89ae.png)

Participant B (looks fine, this is the participant that is refreshing over and over again)
![Screen Shot 2021-07-27 at 1 01 58 PM](https://user-images.githubusercontent.com/362269/127196905-70599e2d-1653-477c-a932-dd2e8c7cb557.png)

**Environment (please complete the following information):**
 - OS: OSX 11.4 (20F71)
 - Browser: Chrome 92.0.4515.107 (Official Build) (x86_64)
 - App Version: tip of master (sha: 356dc2c, release: 0.4.1)
 - SDK Version: 2.14.1 (version currently in package.json for this sample app)
 - Node.js version: 14.5.0

**Additional context**

**Comment by BrandonMathis** [comment]
**Update**

Further research reveals that this limitation was introduced in Chrome 92 (released July 20)

Additional details can be found here:

https://chromium-review.googlesource.com/c/chromium/src/+/2816118

I am attempting to resolve what portion of this sample codebase is causing this issue. My current production application is highly inspired by this example react app codebase so any assistance would be greatly appreciated 😇

**Comment by BrandonMathis** [comment]
Stumbled across this issue on the [twilio-video](https://github.com/twilio/twilio-video.js) which goes into more detail about this issue.

This problem  will likely be resolved once [the PR containing the fix for this issue](https://github.com/twilio/twilio-video.js/pull/1530) is released. This issue can likely be closed once it is verified that the latest version of twilio-video fixes this error.

**Comment by timmydoza** [comment]
Hey @BrandonMathis - thanks for the detailed bug report!

Looks like you have a good grasp of the issue at hand. Chrome 92 introduced a limit on the number WebMediaPlayers, and this is causing the issue you describe. 

For anyone else reading this thread, there is further discussion about this issue in the [twilio-video.js](https://github.com/twilio/twilio-video.js) SDK repo: https://github.com/twilio/twilio-video.js/issues/1528

A new twilio-video.js version has been released with a workaround for the problem (version [2.15.3](https://github.com/twilio/twilio-video.js/releases/tag/2.15.3)), and we have a [pull request up](https://github.com/twilio/twilio-video-app-react/pull/562) to apply the fix to this app, which will be released today.

Thanks for your help on this issue!

**Comment by timmydoza** [comment]
The fix has been released in version 0.4.2 of the app: https://github.com/twilio/twilio-video-app-react/releases/tag/v0.4.2

I'll leave this issue up for a few days for the sake of visibility, just in case anyone else runs into this issue.
